classifications_csv_dump_export.rb

```diff
@@ -0,0 +1,59 @@
+# Manual csv classifications dump
```

Comment:
this file shouldn't be in this PR, please remove

Reply:
doh! `-a` is a dangerous flag to have committed to muscle memory